processor/ratelimitprocessor/local.go

```diff
@@ -59,20 +64,44 @@ func (r *localRateLimiter) RateLimit(ctx context.Context, hits int) error {
 	switch r.cfg.ThrottleBehavior {
 	case ThrottleBehaviorError:
 		if ok := limiter.AllowN(time.Now(), hits); !ok {
+			r.requestTelemetry(ctx, []attribute.KeyValue{
+				telemetry.WithReason(telemetry.TooManyRequests),
+				telemetry.WithDecision("throttled"),
+				telemetry.WithThrottleBehavior("error"),
```

Comment:
I believe we agreed to remove these extra dimensions as they are conveyed by the decision already? Can we remove this. Also the over limit/under limits.

Reply:
Yeah I added them back because if we want to tell the difference between throttled with delay or error, there won't be a way without these fields. Is that the level of details that does not matter?

Reply:
Conclusion: 

- we accept when we can't reach gubernator
- If user is over the limit, we reject with throttled decision
- Under limits, we respond with accepted

Both cases we have the calculated threshold, delay or error does not matter here. Let's keep it simple for now and if we find useful later, we can add it then.